marketplace-ui/src/app/modules/product/product-detail/product-detail-version-action/product-detail-version-action.component.ts

```diff
@@ -138,6 +155,17 @@ export class ProductDetailVersionActionComponent implements AfterViewInit {
       });
   }
 
+
+  getVersionInDesigner(): void {
```

Comment:
I feel that this method will call API whenever the user clicks on the dropdown regardless of the existing versions list.  Please check the current version list size before call api